modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -59,12 +74,7 @@ The channels you need for this procedure are:
 | {centos} 8 | centos8      | centos8-uyuni-client | centos8-appstream
 |===
 
-ifeval::[{uyuni-content} == true]
-include::snippets/add_channels_cli_multiarch.adoc[]
-endif::[]
-ifeval::[{suma-content} == true]
-include::snippets/add_channels_cli.adoc[]
-endif::[]
+include::snippets/add_channels_scc_multiarch.adoc[]
```

Comment:
SCC multiarch on Uyuni for CentOS? No, not right. No SCC for any OS on Uyuni except SLE.

Reply:
You list spacewalk-common-channels on Uyuni for RHEL, CentOS, Oracle, Ubuntu, Debian, openSUSE, and SLE:
 
![image](https://user-images.githubusercontent.com/3914967/98321132-04718680-2030-11eb-9553-f7944f0ea648.png)

The multi-arch was added to CentOS by someone other than me, and I'm not sure of the reasoning behind that, so I left it as is.

Reply:
No, what I don't like at all is the fact that you call it `scc`.

SCC = SUSE Customer Center. Don't use it for `spacewalk-common-channels`.